### ISSUE TITLE
ARIA is insufficient and fails screenreaders

### ISSUE THREAD

**Issue by Volker-E** [nico3333fr/van11y-accessible-hide-show-aria#8]
Your examples look nice, but they are failing the point of ARIA attributes completely. You link to the [excellent inspiration](http://www.oaa-accessibility.org/example/20/), but the [resulting markup](https://van11y.net/downloads/hide-show/demo/index.html) is adding `aria-expanded` on the button instead of the expanded field. You're also not hiding the expandable in any way from screenreaders.

Current output code
```lang=html
<h2 class="js-expandmore" data-hideshow-prefix-class="animated">
  <button class="animated-expandmore__button js-expandmore-button" data-controls="expand_1ugacpb3ta" aria-expanded="true" id="label_expand_1ugacpb3ta" type="button"><span class="animated-expandmore__symbol" aria-hidden="true"></span>Example animated (opened by default)</button>
</h2>
<div class="js-to_expand animated-expandmore__to_expand" data-labelledby="label_expand_1ugacpb3ta" id="expand_1ugacpb3ta">
    <p>Yes, it rocks.</p>
 </div>
```

**Comment by nico3333fr** [comment]
Well not exactly.

> the triggering link or button should indicate to screen reader users whether the collapsable content below is in the expanded or in the collapsed state. The aria-expanded attribute is used for this purpose.

As explained here: https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element 
or https://inclusive-components.design/collapsible-sections/ or here in French https://www.accede-web.com/notices/interface-riche/panneaux-depliants/ 

The hiding is set up using CSS `display: none` or `visibility: hidden`. As you will notice, the script is parametered to use `data-hidden` attribute but could be set up to use `aria-hidden` and other aria stuff depending of your needs. It does not use all of them for one reason: it makes VoiceOver switch to application mode, which is really not intuitive for users, especially for keyboard. As it is a really simple component, it is set up to work in all cases without introducing too much complexity.

Cheers,
Nicolas
